src/length-value.js

```diff
@@ -50,15 +50,23 @@
   LengthValue.prototype.add = function(addedLength) {
     if (this instanceof SimpleLength && addedLength instanceof SimpleLength && this.type == addedLength.type) {
       return this._addSimpleLengths(addedLength);
+    } else if (addedLength instanceof LengthValue) {
+      //ensure both lengths are of type CalcLength before adding
```

Comment:
Nitpick: Comment should start with capital letter, need space after //. Same for subtract.